mappings/net/minecraft/text/ScreenTexts.mapping

```diff
@@ -22,6 +22,7 @@ CLASS net/minecraft/unmapped/C_qveejgrk net/minecraft/text/ScreenTexts
 	METHOD m_qsvbgaen composeToggleText (Lnet/minecraft/unmapped/C_rdaqiwdt;Z)Lnet/minecraft/unmapped/C_npqneive;
 		ARG 0 text
 		ARG 1 value
+	METHOD m_sarnrcsh getSpace ()Lnet/minecraft/unmapped/C_npqneive;
```

Comment:
During my journey around MC's codebase diffs? This would be an utility method like `Text.literal()` (QM) considering it's called `CommonComponents.space()` (MM); Therefore:
```suggestion
	METHOD m_sarnrcsh space ()Lnet/minecraft/unmapped/C_npqneive;
```
yeah, it's pretty much the mutable version of the immutable `ScreenTexts.SPACE` :P
hm, i do wonder if this class will need renaming in the future